backend/init_data/skills/wegent-knowledge/provider.py

```
@@ -0,0 +1,839 @@
+
+        finally:
+            # Cleanup temp directory - always run on success and exceptions
+            try:
+                await sandbox.commands.run(
+                    cmd=["rm", "-rf", temp_dir],
+                    cwd="/home/user",
+                )
+                logger.info(
+                    "[%s] Cleaned up temp_dir=%s", self.__class__.__name__, temp_dir
+                )
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)